cmake/create_test.cmake

```diff
@@ -32,7 +32,6 @@ function(ginkgo_set_test_target_properties test_target_name test_library_suffix)
     else()
         target_link_libraries(${test_target_name} PRIVATE ginkgo_gtest_main${test_library_suffix})
     endif()
-    target_compile_features(${test_target_name} PUBLIC cxx_std_14)
```

Comment:
Is a setting to `${GINKGO_CXX_STANDARD_FEATURE}` not necessary here ? Same question for the test cmake files below.

Reply:
I think it was unnecessary to begin with. The test is linked against ginkgo, so the standard requirement from there is propagated.